zync_maya.py

```diff
@@ -1269,6 +1269,7 @@ def submit(window):
           cmds.setAttr('vraySettings.hideRVOn', 1)
 
           cmds.setAttr('vraySettings.animBatchOnly', 0)
+          cmds.setAttr('defaultRenderGlobals.animation', 1)
```

Comment:
Could you add a comment why we set this? (The same comment that you wrote in the pull request).